acceptance/bin/extract_command_exec_id.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import json
+import re
+import sys
+from pathlib import Path
+
+
+def extract_cmd_exec_id():
+    requests_file = Path("out.requests.txt")
+
+    # Read until we find a complete JSON object
```

Comment:
this is because we pretty-print json, right? Could you add this info to comment?